dbusmock/templates/bluez5.py

```diff
@@ -92,18 +106,22 @@ def AddAdapter(self, device_name, system_name):
         'PairableTimeout': dbus.UInt32(180, variant_level=1),
     }
 
+    if device_name == 'hci1':
+        adapter_properties['Address'] = 'AA:01:02:03:04:05'
```

Comment:
It would be better to create a separate function that "calculated" an address and use that just above. Please do that in a separate commit.